package.json

```diff
@@ -43,6 +43,7 @@
     "eslint": "^8.17.0",
     "eslint-config-particle": "^2.5.0",
     "lodash": "^4.17.21",
+    "nise": "^5.1.4",
```

Comment:
Is this dependency required for this PR? `"nise": "^5.1.4",`

Reply:
There is a downstream dependency issue with `fake-timers`, which doesn't work under Node 12. Forcing `nise` to stay at 5.1.4 fixes this. Only affects dev dependencies/tests.